site/docs/4.3/components/forms.md

```diff
@@ -807,7 +807,7 @@ Custom feedback styles apply custom colors, borders, focus styles, and backgroun
     <div class="col-md-3 mb-3">
       <label for="validationCustom04">State</label>
       <select id="inputState" class="form-control" id="validationCustom04" required>
```

Comment:
per the description just before the example/code block, best to also change the class to `class="custom-select"`

Reply:
remove the redundant `id="inputState"`